docs/manual/src/udl/callback_interfaces.md

```diff
@@ -111,9 +59,17 @@ class KotlinKeychain: Keychain {
         // … elide the implementation.
         return value
     }
-    override fun put(key: String) {
+    override fun put(key: String, value: String) {
         // … elide the implementation.
     }
+
+    override async fun getAsync(key: String) = withContext(coroutineContext) {
+        get(key)
+    }
+
+    override async fun putAsync(key: String, value: String) = withContext(coroutineContext) {
```

Comment:
Nit:
```suggestion
    override suspend fun putAsync(key: String, value: String) = withContext(coroutineContext) {
```

Thinking about it however, doesn't this dance with the `coroutineContext` and `suspend` simply disappear now? (Unless you're intent was to show the kotlin running on a different queue to the `TaskQueue` that was passed in at construction?

Reply:
Good point.  I think you're right, although I'd need to test it to be sure.